dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -376,65 +379,46 @@ Path findClassLocation(String className) {
   static ImmutableMap<String, Path> mapClassToJar(List<Path> jars) throws IOException {
```

Comment:
You could implement this method based on `mapJarToClasses` and avoid some code duplication:

```java
return ImmutableMap.copyOf(
    Maps.transformValues(
        Multimaps.asMap(mapJarToClasses(jars).inverse()), jars -> Iterables.get(jars, 0)));
```

But I have a question: Why is it important to know the _first_ jar that contains a given class, and not the rest? How is this method used?